src/test/kotlin/org/rust/ide/annotator/fixes/ConvertToOwnedTyFixTest.kt

```diff
@@ -0,0 +1,75 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import org.rust.ide.inspections.RsExperimentalChecksInspection
+import org.rust.ide.inspections.RsInspectionsTestBase
+
+class ConvertToOwnedTyFixTest : RsInspectionsTestBase(RsExperimentalChecksInspection()) {
+    override fun getProjectDescriptor() = WithStdlibRustProjectDescriptor
+
+    fun `test B is Owned type of A`() = checkFixByText("Convert to owned type using `ToOwned` trait","""
+        use std::borrow::Borrow;
+
+        struct A{}
+        struct B{}
+
+        impl Borrow<A> for B { fn borrow(&self) -> &A { &A {} } }
+
+        impl ToOwned for A {
+            type Owned = B;
+            fn to_owned(&self) -> B { B {} }
```

Comment:
AFAIK if you want to instantiate a struct without fields, you should just write `B` without braces.

Reply:
only if `B` is also declared as `struct B;`, if it's declared as `struct B {}` it has to be instantiated with `B {}` too, don't really know why